lib/js/components/RichList/RichListItemBasic/RichListItemBasic.vue

```diff
@@ -0,0 +1,172 @@
+<template>
```

Comment:
![image](https://github.com/bethinkpl/design-system/assets/11304259/a15bed28-642c-4b9f-a388-3f903e960f8a)
I think trailing slot is too aggressive when it comes to space allocation

Reply:
![image](https://github.com/bethinkpl/design-system/assets/11304259/8b1bb573-658f-4698-8268-02623de031cc)
It is weird on RichListItem too

Reply:
But I am not sure if it is a bug, or we just shouldn't place such content there

Reply:
![image](https://github.com/bethinkpl/design-system/assets/11304259/1f4b282a-1d8c-49ba-9852-85a453b376d7)
Again RichListItem

Reply:
![image](https://github.com/bethinkpl/design-system/assets/11304259/0a1fc5d4-b146-49dd-8460-b6e79f6f1fac)
content slot still look a bit too high, or icons too low